lib/styler/config.ex

```diff
@@ -145,6 +161,11 @@ defmodule Styler.Config do
       {ParenthesesOnZeroArityDefs, opts}, acc when is_list(opts) ->
         Map.put(acc, :zero_arity_parens, opts[:parens])
 
+      {PipeChainStart, opts}, acc when is_list(opts) ->
+        Map.put(acc, :pipe_chain_start_flag, true)
+        Map.put(acc, :pipe_chain_start_excluded_functions, opts[:excluded_functions])
+        Map.put(acc, :pipe_chain_start_excluded_argument_types, opts[:excluded_argument_types])
```

Comment:
```suggestion
        Map.put(acc, :pipe_chain_start_flag, true)
        |> Map.put(:pipe_chain_start_excluded_functions, opts[:excluded_functions])
        |> Map.put(:pipe_chain_start_excluded_argument_types, opts[:excluded_argument_types])
```

Need to pipe or resign the acc otherwise only the last value here is actually put into it and the other 2 lines would be unmatched returns

Reply:
Ah nice catch. I thought I had fixed that. Not sure what happened. Thanks!